app/vmalert/Makefile

```diff
@@ -1,6 +1,9 @@
 # All these commands must run from repository root.
 
-vmalert:
+templates-gen: install-qtc
+	qtc	-dir=./app/vmalert
+
+vmalert: templates-gen
```

Comment:
The `templates-gen` dependency isn't needed here, since `make vmalert` must build the code from already generated and stored in the repository Go code for templates.

Reply:
This comment has been addressed in e6c9869d86b444bb05affed30884ac38bd7e835d